uint8 id
geometry_msgs/Point32 position
geometry_msgs/Point32 velocity
